README.md

```diff
@@ -14,7 +14,7 @@ You can run and edit the algorithms or contribute to them using Gitpod.io, a fre
 These implementations are for learning purposes. The implementations may be less efficient than the Java standard library.
 
 ## Contribution Guidelines
-Read our [Contribution Guidelines](CONTRIBUTING.md) before you contribute.
+Read our [Contribution Guidelines](CONTRIBUTING.md) before you contribute anything.
```

Comment:
These type of minor changes in readme during hacktoberfest are considered as spams.